src/http.test.ts

```diff
@@ -59,6 +59,52 @@ describe('http', () => {
         expect(response.status).to.equal(404);
     });
 
+    it('route parameters', async () => {
+        const funcUrl = getFuncUrl('httpTriggerRouteParams');
+        const response = await fetch(`${funcUrl}/testName`);
+        const body = await response.json();
+        expect(body).to.deep.equal({ name: 'testName' });
+        expect(response.status).to.equal(200);
+    });
+
+    it('headers', async () => {
+        const funcUrl = getFuncUrl('httpTriggerHeaders');
+        const response = await fetch(funcUrl, { headers: { TESTHEADER: 'TestValue' } });
+        const body = await response.json();
+        expect(body).to.deep.equal({
+            accept: '*/*',
```

Comment:
Do we need to check the whole body? Or can we just check the testheader property?

Reply:
I wanted to test all headers to make sure we have consistent behavior for stream vs non-stream. This is one of the things I fixed in my PR in the library because before they weren't matching: https://github.com/Azure/azure-functions-nodejs-library/pull/238

### Before my fix (stream-only)

```json
{
    "accept": "*/*"
    "accept-encoding": "gzip,deflate"
    "host": "localhost:57183"
    "testheader": "TestValue"
    "traceparent": "00-698606ffda1cbb22ea7a3320d846f9a7-1660cc994902dfe2-00"
    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"
    "x-forwarded-for": "127.0.0.1"
    "x-forwarded-host": "127.0.0.1:7071"
    "x-forwarded-proto": "http"
    "x-ms-invocation-id": "7feef90c-0e86-49c6-8db7-3190bc2bb736"
}
```

### After my fix (stream and non-stream now match)

```json
{
    "accept": "*/*"
    "accept-encoding": "gzip,deflate"
    "connection": "close"
    "host": "127.0.0.1:7071"
    "testheader": "TestValue"
    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"
}
```